index.html

```diff
@@ -2,8 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/img/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/img/favicon-32x32.png">
```

Comment:
Prosím pohlídat formátování kódu

Reply:
Návrh: nebylo by hezčí mít pro ikonky samostatnou podsložku?

Reply:
Zatím to necháme takto v HTML a případně časem to můžeme dle potřeby upravit.